core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -129,13 +161,32 @@ export class TranslationModalComponent {
     return this.HTML_SCHEMA;
   }
 
-  onContentClick(event: MouseEvent): void {
+  onContentClick(event: MouseEvent): boolean | void {
+    if (this.validateParagraphCopy(event)) {
+      return this.triedToCopyText = true;
+    }
+    this.triedToCopyText = false;
     if (this.isCopyModeActive()) {
       event.stopPropagation();
     }
     this.ckEditorCopyContentService.broadcastCopy(event.target as HTMLElement);
   }
 
+  validateParagraphCopy($event: MouseEvent): boolean {
```

Comment:
```suggestion
  triedToCopyParagraph($event: MouseEvent): boolean {
```

Reply:
I adjusted this as `userTriedToCopyParagraph` as already a variable is named as `triedToCopyParagraph `

Reply:
I think it makes more sense to name this method `triedToCopyParagraph` and the variable `hadCopyParagraphError` to be consistent with the other error variable names.

Reply:
Done